h2/src/test/org/h2/test/db/TestFullText.java

```diff
@@ -50,10 +50,6 @@ public static void main(String... a) throws Exception {
 
     @Override
     public void test() throws Exception {
-        if (config.multiThreaded) {
-            // It is even mentioned in the docs that this is not supported
-            return;
-        }
         testUuidPrimaryKey(false);
```

Comment:
Do you want me to just drop the last paragraph under tutorial.html#fulltext about lucene not being synchronized internally? Will do that. Native implementation thread safety is not even mentioned there. Any other places in the docs?